packages/atomic/src/components.d.ts

```diff
@@ -60,12 +60,12 @@ export namespace Components {
         "updateMessage": (region: string, message: string, assertive: boolean) => Promise<void>;
     }
     /**
-     * @beta - This component is part of the automatic facets feature.
-     * Automatic facets are currently in beta testing and should be available soon.
-     * An automatic facet is a special type of facet generated by the automatic facets feature. Unlike traditional facets that
-     * need to be explicitly defined and requested in the search query, automatic facets are dynamically generated by the index
-     * in response to the search query. It's important to note that this component should not be used directly. It is internally used
-     * by the `atomic-automatic-facet-generator` component to automatically render updated facets.
+     * An automatic facet is a special type of facet generated by the automatic facets feature.
+     * Unlike traditional facets that need to be explicitly defined and requested in the search query, automatic facets are dynamically generated by the index
+     * in response to the search query.
+     * It's important to note that this component is not typically used directly.
```

Comment:
I personally find this more clear. I was uncertain as to what it meant for a component to be used directly vs indirectly. Moreover, I think this should go _**after**_ the the next sentence that explains that it's used internally by the `atomic-automatic-facet-generator` component.
```suggestion
     * It's important to note that this component is not typically used on its own.
```